assets/src/stories-editor/constants.js

```diff
@@ -31,7 +31,12 @@ export const STORY_PAGE_INNER_WIDTH = 328;
 export const STORY_PAGE_INNER_HEIGHT = 553;
 
 export const MIN_BLOCK_WIDTH = 40;
-export const MIN_BLOCK_HEIGHT = 30;
+export const MIN_BLOCK_HEIGHTS = {
```

Comment:
Can we somehow consolidate this with `getDefaultMinimumBlockHeight`?

Reply:
Hmm, I'm not sure because one of these is used for the initial default height and the other one is for the minimum allowed resizing. The minimum allowed heights don't necessarily make sense as the default heights. Thoughts?

Reply:
Fair enough. A mention in the docs would be helpful though.